addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -139,9 +143,11 @@ DuplicatePageDialog.template = xml`
 </WebsiteDialog>
 `;
 DuplicatePageDialog.props = {
+    duplicate: {type: Function, optional: true},
     onDuplicate: {type: Function, optional: true},
     close: Function,
     pageId: Number,
+    pageIds: {type: [Number], optional: true},
```

Comment:
I am not familiar with props definition, but this looked louche. I checked how other props are defined in our codebase and `[]` symbols are actually used to explain it should be of any type inside that list, but it does not mean it should be an array (of type integer).

So here, it is the same as `type: Number` without the `[]`, as you only added one possibility in the list. And if you wrote `[Number, Function]`, it would have meant that it should either be a Number **or** a Function.

I guess it should be `type: Object`?

You can actually enter debug mode and try to duplicate a page, you will see:
![Uploading image.png…]()

Reply:
Probably also add `element: Number` or something like that:

> element: if the type was Array, then the element key describes the type of each element in the array. If it is not set, then we only validate the array, not its elements

https://github.com/odoo/owl/blob/master/doc/reference/props.md